vendor/github.com/ActiveState/graphql/graphql.go

```diff
@@ -48,6 +49,9 @@ type Client struct {
 	httpClient       *http.Client
 	useMultipartForm bool
 
+	// closeReq will close the request body immediately allowing for reuse of client
+	closeReq bool
```

Comment:
You seem to have forked off of a different version than we were on. Please fork off of the exact version we were using. This changeset should be as small as possible.